plugins/inputs/xtremio/README.md

```diff
@@ -0,0 +1,108 @@
+# XtremIO Input Plugin
+
+The `xtremio` plugin gathers metrics from a Dell EMC XtremIO Storage Array's V3 Rest API. Documentation can be found [here](https://dl.dell.com/content/docu96624_xtremio-storage-array-x1-and-x2-cluster-types-with-xms-6-3-0-to-6-3-3-and-xios-4-0-15-to-4-0-31-and-6-0-0-to-6-3-3-restful-api-3-x-guide.pdf?language=en_us)
+### Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage xtremio`.
+
+```toml
+[[inputs.xtremio]]
+  ## XtremIO Username
+  username = "user1" # required
+  ## XtremIO Password
+  password = "pass123" # required
+  ## XtremIO User Interface Endpoint
+  url = "https://xtremio.example.com/" # required
```

Comment:
Usually we start with the URL or server. You can also shorten some comments for self-explaining settings
```suggestion
  ## XtremIO User Interface Endpoint
  url = "https://xtremio.example.com/" # required

    ## Credentials
  username = "user1"
  password = "pass123"
```